src/main/java/org/javarosa/core/model/instance/DataInstance.java

```diff
@@ -23,13 +23,15 @@
  *
  */
 public abstract class DataInstance<T extends AbstractTreeElement<T>> implements Persistable {
-
-
     /** The integer Id of the model */
     private int recordid = -1;
 
-    /** The name for this data model */
+    /**
+     * A unique name for this instance. The value of the id attribute in the case of a secondary instance or the title
```

Comment:
It's strange to describe the how this field is used in this comment. However, I couldn't find a better place for it and it seemed valuable to me for making sense of debug contents. I can remove it if others disagree, though.